### ISSUE TITLE
sls invoke local passes null environment variables as "null" string to handlers

### ISSUE THREAD

**Issue by yyamano** [serverless/serverless#9261]
sls invoke local passes null environment variables as "null" string to handlers.

With the following definition:

```
  environment:
    FOO: ${env:FOO,null} 
```

I get the message as below, but I expected to get "Foo is null".

```
2021-04-08 14:51:51  INFO  Handler:23 - Foo is "null"
```

Here is a fragment of my handler code:

```
                String foo = System.getenv("FOO");
                if (foo == null) {
                        LOG.info("Foo is null");
                } else {
                        LOG.info("Foo is \"" + foo + "\"");
                }
```

The project is generated by "create -t aws-java-maven"  and applied the following change.

```
diff -u -r aws-java-maven.orig/serverless.yml aws-java-maven/serverless.yml
--- aws-java-maven.orig/serverless.yml	2021-04-08 14:32:02.000000000 +0900
+++ aws-java-maven/serverless.yml	2021-04-08 14:03:27.000000000 +0900
@@ -11,7 +11,7 @@
 #
 # Happy Coding!
 
-service: aws-java-maven.orig
+service: aws-java-maven
 # app and org for use with dashboard.serverless.com
 #app: your-app-name
 #org: your-org-name
@@ -46,8 +46,8 @@
 #            - "/*"
 
 # you can define service wide environment variables here
-#  environment:
-#    variable1: value1
+  environment:
+    FOO: ${env:FOO,null} 
 
 # you can add packaging information here
 package:
diff -u -r aws-java-maven.orig/src/main/java/com/serverless/Handler.java aws-java-maven/src/main/java/com/serverless/Handler.java
--- aws-java-maven.orig/src/main/java/com/serverless/Handler.java	2021-04-08 14:32:02.000000000 +0900
+++ aws-java-maven/src/main/java/com/serverless/Handler.java	2021-04-08 14:10:27.000000000 +0900
@@ -16,6 +16,12 @@
 	@Override
 	public ApiGatewayResponse handleRequest(Map<String, Object> input, Context context) {
 		LOG.info("received: {}", input);
+		String foo = System.getenv("FOO");
+		if (foo == null) {
+			LOG.info("Foo is null");
+		} else {
+			LOG.info("Foo is \"" + foo + "\"");
+		}
 		Response responseBody = new Response("Go Serverless v1.x! Your function executed successfully!", input);
 		return ApiGatewayResponse.builder()
 				.setStatusCode(200)
```

<details>
<summary><code>serverless.yml</code></summary>

```yaml
# Welcome to Serverless!
#
# This file is the main config file for your service.
# It's very minimal at this point and uses default values.
# You can always add more config options for more control.
# We've included some commented out config examples here.
# Just uncomment any of them to get that config option.
#
# For full config options, check the docs:
#    docs.serverless.com
#
# Happy Coding!

service: aws-java-maven
# app and org for use with dashboard.serverless.com
#app: your-app-name
#org: your-org-name

# You can pin your service to only deploy with a specific Serverless version
# Check out our docs for more details
frameworkVersion: '2'

provider:
  name: aws
  runtime: java8
  lambdaHashingVersion: 20201221

# you can overwrite defaults here
#  stage: dev
#  region: us-east-1

# you can add statements to the Lambda function's IAM Role here
#  iamRoleStatements:
#    - Effect: "Allow"
#      Action:
#        - "s3:ListBucket"
#      Resource: { "Fn::Join" : ["", ["arn:aws:s3:::", { "Ref" : "ServerlessDeploymentBucket" } ] ]  }
#    - Effect: "Allow"
#      Action:
#        - "s3:PutObject"
#      Resource:
#        Fn::Join:
#          - ""
#          - - "arn:aws:s3:::"
#            - "Ref" : "ServerlessDeploymentBucket"
#            - "/*"

# you can define service wide environment variables here
  environment:
    FOO: ${env:FOO,null} 

# you can add packaging information here
package:
  artifact: target/hello-dev.jar

functions:
  hello:
    handler: com.serverless.Handler
#    The following are a few example events you can configure
#    NOTE: Please make sure to change your handler code to work with those events
#    Check the event documentation for details
#    events:
#      - httpApi:
#          path: /users/create
#          method: get
#      - websocket: $connect
#      - s3: ${env:BUCKET}
#      - schedule: rate(10 minutes)
#      - sns: greeter-topic
#      - stream: arn:aws:dynamodb:region:XXXXXX:table/foo/stream/1970-01-01T00:00:00.000
#      - alexaSkill: amzn1.ask.skill.xx-xx-xx-xx
#      - alexaSmartHome: amzn1.ask.skill.xx-xx-xx-xx
#      - iot:
#          sql: "SELECT * FROM 'some_topic'"
#      - cloudwatchEvent:
#          event:
#            source:
#              - "aws.ec2"
#            detail-type:
#              - "EC2 Instance State-change Notification"
#            detail:
#              state:
#                - pending
#      - cloudwatchLog: '/aws/lambda/hello'
#      - cognitoUserPool:
#          pool: MyUserPool
#          trigger: PreSignUp
#      - alb:
#          listenerArn: arn:aws:elasticloadbalancing:us-east-1:XXXXXX:listener/app/my-load-balancer/50dc6c495c0c9188/
#          priority: 1
#          conditions:
#            host: example.com
#            path: /hello

#    Define function environment variables here
#    environment:
#      variable2: value2

# you can add CloudFormation resource templates here
#resources:
#  Resources:
#    NewResource:
#      Type: AWS::S3::Bucket
#      Properties:
#        BucketName: my-new-bucket
#  Outputs:
#     NewOutput:
#       Description: "Description for the output"
#       Value: "Some output value"
```

</details>

<details>
<summary><b><code>sls  invoke local -f hello</code> output</b></summary>

```
% sls  invoke local -f hello
Serverless: In order to get human-readable output, please implement "toString()" method of your "ApiGatewayResponse" object.
2021-04-08 14:45:14  INFO  Handler:18 - received: {}

2021-04-08 14:45:14  INFO  Handler:23 - Foo is "null"

com.serverless.ApiGatewayResponse@4009e306
```

</details>

<b>Installed version</b>

```
Framework Core: 2.34.0
Plugin: 4.5.3
SDK: 4.2.2
Components: 3.8.2
```
The latest version https://github.com/serverless/serverless/commit/7072795a50ce87879089d41335641ac37d4abd09 from the repository.

**Comment by yyamano** [comment]
@pgrzesik Could you close the issue? 
It was fixed by #9263. My fault.

**Comment by pgrzesik** [comment]
Thank you @yyamano :bow:
